pkg/auth/model/model.go

```diff
@@ -122,3 +124,114 @@ type CredentialKeys struct {
 	AccessKeyID     string `json:"access_key_id"`
 	SecretAccessKey string `json:"secret_access_key"`
 }
+
+func userFromProto(pb *UserData) *User {
+	return &User{
+		ID:                pb.ID,
+		CreatedAt:         pb.CreatedAt.AsTime(),
+		Username:          pb.Username,
+		FriendlyName:      &pb.FriendlyName,
+		Email:             &pb.Email,
+		EncryptedPassword: pb.EncryptedPassword,
+	}
+}
+
+func protoFromUser(u *User) *UserData {
+	return &UserData{
+		ID:                u.ID,
+		CreatedAt:         timestamppb.New(u.CreatedAt),
```

Comment:
No - and when we created the data we need to use time.UTC()

Reply:
I don't understand. timestamps are created in many places in lakeFS, several of those in the auth package. Are we expecting to pass the responsibility of using the right TZ to every line of code creating timestamp to keep proto conversion correct? I think this layer of golang<>proto conversion needs to handle that, otherwise we'll be chasing our tail with TZ bugs.

Reply:
protobuf's timestamppb currently doesn't hold localization data, so I don't see a way of getting around it.